internal/engine/analytics_reporter.go

```diff
@@ -91,4 +91,14 @@ func (ar *AnalyticsReporter) report() {
 	stats["tiltfile.error"] = tiltfileIsInError
 
 	ar.a.Incr("up.running", stats)
+
+	for index := 0; index < st.OpenLogCount; index++ {
+		ar.a.Incr("ui.interactions.log_open", map[string]string{})
+	}
+	st.OpenLogCount = 0
```

Comment:
also, for future reference, this assignment is a no-op. subscribers don't have write access to the EngineState.  `st` isn't a pointer. This mutates a copy of the engine state, which is probably not what you intended.